docs/setup-cluster/workspaces-rpools.rst

```diff
@@ -28,7 +28,9 @@ workspace.
 
 In addition, you can set a bound resource pool as the default compute or auxiliary pool for the
 workspace. If a user leaves the resource pool configuration option blank for their task, workloads
-will be sent to the default compute or auxiliary pool.
+will be sent to the default compute or auxiliary pool. If the resource pool does not have a default
```

Comment:
Do you mean If the workspace instead of If the resource pool does not have a default...?

Reply:
ahhhh good catch